### ISSUE TITLE
Used latest Spark Image to create a Standalone cluster - error of Initial Job didn't accept any resources

### ISSUE THREAD

**Issue by kiran-jayaram** [bitnami/charts#12687]
### Name and Version

bitnami/Spark:3.2.1 (but happens on 3.3 as well

### What steps will reproduce the bug?

Created a network with a single worker and but everytime I start it up, I am able to submit a job but I can't get it to execute as it's not able to find resources.

### What is the expected behavior?

On my local Spark, I am logging in with the --master spark://localhost:7077 and then trying to run a basic job

### What do you see instead?

![image](https://user-images.githubusercontent.com/95016070/192393172-7687e671-6fb6-4336-91fd-897125b35179.png)

### Additional information

The below is the compose_file

```
version: '2'

networks:
  netw:

services:
  spark-master:
    image: docker.io/bitnami/spark:3.2.1

    environment:
      - SPARK_MODE=master
      - SPARK_RPC_AUTHENTICATION_ENABLED=no
      - SPARK_RPC_ENCRYPTION_ENABLED=no
      - SPARK_LOCAL_STORAGE_ENCRYPTION_ENABLED=no
      - SPARK_SSL_ENABLED=no
      - SPARK_MASTER_PORT=7077
      - SPARK_MASTER_WEBUI_PORT=8080      

    ports:
      - '8080:8080'
      - '7077:7077'
      - '4040:4040'
      - '6066:6066'
    networks:
      - netw

  spark-worker:
    image: docker.io/bitnami/spark:3.2.1
    environment:
      - SPARK_MODE=worker
      - SPARK_MASTER_URL=spark://spark-master:7077      
      - SPARK_WORKER_MEMORY=8G
      - SPARK_WORKER_CORES=24      
      - SPARK_WORKER_PORT=40001
      - SPARK_WORKER_WEBUI_PORT=8081
      - SPARK_RPC_AUTHENTICATION_ENABLED=no
      - SPARK_RPC_ENCRYPTION_ENABLED=no
      - SPARK_LOCAL_STORAGE_ENCRYPTION_ENABLED=no
      - SPARK_SSL_ENABLED=no

    ports:
      - '40001:40001'
      - '8081:8081'

    networks:
      - netw

    depends_on:
      - spark-master
```

**Comment by marcosbc** [comment]
Hi @kiran-jayaram, were you able to reproduce this issue with the chart?

If not, please note that the location where we track issues related to docker-compose usage is the [bitnami/containers](https://github.com/bitnami/containers/issues) repo.

If you open the issue there, I would recommend you to try reproducing the issue with the provided [docker-compose.yml file](https://github.com/bitnami/containers/blob/main/bitnami/spark/docker-compose.yml). In addition, the logs mention that the UI might be able to provide additional info, which could be useful for understanding what is happening.

**Comment by kiran-jayaram** [comment]
@marcosbc  - thanks for the reply.
I had a peek into my master and worker logs. It seems to work when I open a pyspark session within one of the 2 (worker/master)
i.e. pyspark --master spark://spark:7077

however, when I try this from outside the container - i.e. a local spark installed on my windows desktop -> 
e,g, 
C:\spark> pyspark --master spark://localhost:7077 
then I get 
![image](https://user-images.githubusercontent.com/95016070/193166870-5baedcba-3ebf-4802-a28a-d5b9b0548be2.png)

Basically, it seems that executor is not trying to register and fails. and it keeps hanging.

**Comment by marcosbc** [comment]
Hi @kiran-jayaram, I remember there being issues with Spark and other Java applications using Docker, due to the proxy that it enforces. If you install the Helm chart, you should not find these issues.

**Comment by kiran-jayaram** [comment]
Thanks @marcosbc - it's a bit tricky that. I was able to get it working on local using my local the bindadress that is specified in my etc/hosts file. However, it behaves differently with my corporate VPN, corporate LAN and corporate WiFi.
I am trying to get this working on another labbox that we have that's different network - not much luck so far

**Comment by marcosbc** [comment]
@kiran-jayaram Have you tried options such as minikube? That should work in your local environment.
